drivers/hmis/lib/hmis_util/json_forms.rb

```diff
@@ -260,10 +260,10 @@ def load_definition(form_definition:, identifier:, role:, title: nil)
         identifier: identifier,
         role: role,
         version: 0,
-        status: Hmis::Form::Definition::DRAFT,
-      ).first_or_create!(title: title || role.to_s.humanize)
+      ).first_or_create!(title: title || role.to_s.humanize, status: Hmis::Form::Definition::PUBLISHED)
```

Comment:
This is fine but looks like the record is saved twice (create and then save). Seems like we could just make a new record and let the save down on 267 handle it
```suggestion
      ).first_or_initialize(title: title || role.to_s.humanize)
```